docs/api-docs/customers/current-customer-api.md

```diff
@@ -1,60 +1,30 @@
+<!-- theme: info -->
+> #### API account notes
+> - This endpoint requires **app API account** credentials. For more information about generating accounts, consult our [Guide to API Accounts](/api-docs/getting-started/authentication/rest-api-authentication#app-api-accounts).
+> - The app you create doesn't need to be installed or published on a store, and you don't need to generate access tokens. All you need are the client ID and client secret. See the section on [client ID-based authentication](/api-docs/getting-started/authentication#client-id) in the Authentication tutorial.
 
-To address this need, BigCommerce provides a Current Customer endpoint that your app can access via JavaScript on the storefront. This endpoint allows a remote application, such as a third-party subscription billing app, to return a JWT with identifying customer details. The information is signed with your [OAuth client secret](/api-docs/getting-started/basics/authentication#authentication_client-id-secret).
 
-<!-- theme: info -->
-> #### Note
-> - An app client ID is required in requests to `/customer/current.jwt`.
-> - To generate an app client ID, create an app in the [BigCommerce Developer Portal](https://devtools.bigcommerce.com/).
-> - Use the app's secret to validate the signature on the JWT.
-> - The app doesn't need to be installed or published on a store to use the client ID to get the JWT.
-
-
-
-## Example JavaScript
-
-Below is an example JavaScript code snippet that will access this JWT. To test the JWT functionality, you can install this JavaScript on your sandbox BigCommerce store. You must include your application's client ID in the request to identify the requesting application.
-
-```html
-<script type="text/javascript">
-  function customerJWT() {
-    var appClientId = '**BC_CLIENT_ID**'; // TODO: Fill this in with your app's client ID
-    var xmlhttp = new XMLHttpRequest();
-    xmlhttp.onreadystatechange = function () {
-      if (xmlhttp.readyState == 4) {
-        if (xmlhttp.status == 200) {
-          alert('Customer JWT:\n' + xmlhttp.responseText);
-        } else if (xmlhttp.status == 404) {
-          alert('Not logged in!');
-        } else {
-          alert('Something went wrong');
-        }
-      }
-    };
-    xmlhttp.open(
-      'GET',
-      '/customer/current.jwt?app_client_id=' + appClientId,
-      true
-    );
-    xmlhttp.send();
-  }
-  customerJWT();
-</script>
-```
+## Request
+
+To test this endpoint, save this [Current Customer API example request](/api-docs/getting-started/authentication#current-customer-api-example-request) JavaScript snippet in the **Script Manager** of one of your sandbox store's Stencil storefronts. Include your app API account's client ID as the value of the `app_client_id` query parameter.

```

Comment:
I would change to: "To test the Current Customer endpoint..." Consider adding either a link to Script Manager docs or a screenshot

Reply:
the issue with screenshots of script manager is that it's different in MSF vs SSF. the DevDocs we have about scripts are about the API. I'm suggesting a link to KB for the UI docs on script manager, which should be kept up to date better than this.

```suggestion
To test this endpoint, save this [Current Customer API example request](/api-docs/getting-started/authentication#current-customer-api-example-request) JavaScript snippet in the storefront's **Script Manager**, located in the store control panel. Include your app API account's client ID as the value of the `app_client_id` query parameter. 

For more about adding scripts with the UI, see our support article on [Using Script Manager](https://support.bigcommerce.com/s/article/Using-Script-Manager).

```